targets/f7/api_symbols.csv

```diff
@@ -2040,6 +2041,7 @@ Function,-,initstate,char*,"unsigned, char*, size_t"
 Function,+,input_get_key_name,const char*,InputKey
 Function,+,input_get_type_name,const char*,InputType
 Function,-,iprintf,int,"const char*, ..."
+Function,+,is_weak_prng_nonce,_Bool,uint32_t
```

Comment:
We need to prefix all symbols in crypto1 package if symbols are exported to api symbol table